### PR TITLE
fix: cc-0088 price value NaN at uat and production

### PR DESCRIPTION
**Pull request by Beryl-Zhou**
fix: price display 0.00 at default page

### CODE REVIEW

File: src/components/PriceBar/TileColorBoard.tsx [hunk 1]
```diff
@@ -9,37 +9,51 @@ const TileColorBoard: React.FC = () => {
...
     if (courtList) {
-      installPrice += courtList.installationPrice / 100;
+      priceDetails.installPrice += courtList.installationPrice / 100;
     }
-    const price = tilePrice + deliveryPrice + installPrice;
-    // check price format, type transfer to string
+  };
+
+  useEffect(() => {
+    if (data == undefined) return;
```
Comment (Fuchih) [inline src/components/PriceBar/TileColorBoard.tsx:48]: Why you use "=="

File: src/components/PriceBar/TileColorBoard.tsx [hunk 2]
```diff
@@ -9,37 +9,51 @@ const TileColorBoard: React.FC = () => {
...
+    calculateTile();
+    calculateDelivery();
+    calculateInstallation();
+    console.log(priceDetails);
```
Comment (Fuchih) [inline src/components/PriceBar/TileColorBoard.tsx:52]: Do we need to output a message to the console?

File: src/components/PriceBar/TileColorBoard.tsx [hunk 3]
```diff
@@ -9,37 +9,50 @@ const TileColorBoard: React.FC = () => {
   const tileBlocks = useStoreSelector((state) => state.priceBar.blocks);
   const courts = useStoreSelector((state) => state.courtSize);
   const { data } = useGetPriceQuery(0);
-  const [useTotalPrice, setTotalPrice] = useState<string>("0.00");
+  const priceList = data?.find((item: IPriceCalculation) => !item.isDeleted);
+  const [useTotalPrice, setTotalPrice] = useState<string>("Loading...");
```
Comment (tatoMa) [inline src/components/PriceBar/TileColorBoard.tsx:13]: It is not acceptable to set a "loading" string into an apparent number variable. Please divide them into 2 variables and use conditions to render the loading text or price number.

Comment (Beryl-Zhou) [reply src/components/PriceBar/TileColorBoard.tsx:13]: actually, the final calculated result will be converted to string type to adjust the price format "X.X0"

**Pull request closed without merging**